vortex-datafusion/src/persistent/opener.rs

```diff
@@ -66,36 +68,36 @@ impl VortexFileOpener {
             projection,
             filter,
             file_layout_cache,
-            config,
+            projected_arrow_schema,
         })
     }
 }
 
 impl FileOpener for VortexFileOpener {
     fn open(&self, file_meta: FileMeta) -> DFResult<FileOpenFuture> {
-        let this = self.clone();
```

Comment:
I was profiling clickbench q0/q1 and saw the clones here, so we now only clone the things that are actually used inside the stream